src/components/weather/DailyWeather.tsx

```diff
@@ -21,7 +22,7 @@ type Props = {
   };
 };
 
-export const DailyWeather = ({ date, icon, temperatures }: Props) => {
+export const DailyWeather = ({ date, desctription, icon, temperatures }: Props) => {
```

Comment:
typo in this file! `desctription` -> `description`

Reply:
Upps sorry 🤦‍♂️ I will update immediately.